pywbem_mock/_wbemconnection_mock.py

```diff
@@ -995,6 +1012,104 @@ def _display_objects(obj_type, object_repo, namespace, cmt_begin, cmt_end,
                     else:
                         _uprint(dest, obj.tomof())
 
+    def register_provider(self, namespaces, classnames, provider_type,
+                          provider):
+        """
+        Register the provider object for namespace and class. Registering a
+        provider tells the FakedWBEMConnection that the provider implementation
+        provided with this call is to be executed as the request response
+        method for the namespace and class defined in lieu of the default
+        provider.
+
+        Providers can only be registered for request response methods
+        CreateInstance and DeleteInstance and InvokeMethod
+
+        Parameters:
+
+          namespace (:term:`string or :class:`py:list` of :term:`string):
```

Comment:
Change `namespace` to `namespaces` to match the actual parameter name.

Reply:
DONE